### PR TITLE
We can now use eth-proof 2.1.4 directly

### PR DESCRIPTION
**Pull request by bstr156**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,10 +3,18 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import SysethereumDApp from './SysethereumDApp';
 import * as serviceWorker from './serviceWorker';
+import TagManager from 'react-gtm-module';
+
+const tagManagerArgs = {
```
Comment (willyko) [inline src/index.js:8]: who's google tag is this?

File: src/config.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
   class CONFIGURATION {
     constructor() {
-      this.testnet = true;
-      this.agentURL = "bridge-testnet.syscoin.org";
+      this.testnet = false;
```
Comment (sidhujag) [inline src/config.js:3]: should be true for testnet branch

Comment (sidhujag) [reply src/config.js:3]: all the other details should be testnet based too

**Comment by sidhujag** [comment]
does this make more sense? https://github.com/syscoin/sysethereum-dapp/commit/cbac38eefc5a3e4ac036f56ee931723c16f138a4

**Comment by bstr156** [comment]
> does this make more sense? [cbac38e](https://github.com/syscoin/sysethereum-dapp/commit/cbac38eefc5a3e4ac036f56ee931723c16f138a4)

Yes.  Accidentally submitted pull request from my fork's master to testnet, instead of from testnet.  Will be mindful going forward.

**Pull request closed without merging**